contrib/test/run_test_vectors.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -f
+
+# this assumes fd_ledger has already been built
+
+set -x
+
+OBJDIR=${OBJDIR:-build/native/gcc}
+
+rm    -rf $LOG_PATH
+mkdir -pv $LOG_PATH
+
+mkdir -p dump
+
+if [ ! -d dump/test-vectors ]; then
+  cd dump
+  git clone --depth=1 -q https://github.com/firedancer-io/test-vectors.git
```

Comment:
because sub-modules are a pit of despair and pain and since our test vectors are not tied to specific versions of FD, there is no reason to use sub-modules.  Heck, everybody HATES the submodule use of the runtime tests